--- extract/um/src/utility/qxreconf/rcf_setup_lsm_out_mod.F90	2019-09-10 14:10:36.000000000 +1000
+++ extract/um/src/utility/qxreconf/rcf_setup_lsm_out_mod.F90	2020-02-07 14:08:23.000000000 +1100
@@ -161,0 +162,2 @@
+use external_spiral_circle_mod
+use gather_saw_mod
@@ -300 +302 @@
-    CALL Gather_Field( local_lsm_out, glob_lsm_out,                            &
+    CALL Gather_Field_saw( local_lsm_out, glob_lsm_out,                            &
@@ -629 +631 @@
-        CALL rcf_spiral_circle_s(glob_lsm_out, index_targ_sea,                 &
+        CALL read_spiral_circle_s('sea.nc',glob_lsm_out, index_targ_sea,                 &
@@ -638 +640 @@
-        CALL rcf_spiral_circle_s(glob_lsm_out, index_targ_land,                &
+        CALL read_spiral_circle_s('land_unc.nc', glob_lsm_out, index_targ_land,                &
@@ -647 +649 @@
-        CALL rcf_spiral_circle_s(glob_lsm_out, index_targ_land,                &
+        CALL read_spiral_circle_s('land_con.nc',glob_lsm_out, index_targ_land,                &
--- fcm-make2.cfg	2019-12-05 16:32:56.000000000 +1100
+++ fcm-make2.cfg	2020-01-22 10:43:19.000000000 +1100
@@ -148 +148 @@
-build-recon.prop{no-dep.f.module} = mpl gcom_mod eccodes netcdf   
+build-recon.prop{no-dep.f.module} = mpl gcom_mod eccodes netcdf   mpi_f08
